gapis/api/gles/synthetic.api

```diff
@@ -38,6 +38,10 @@ cmd void replayChangeBackbuffer(u32     rendererID,
 @synthetic
 cmd EGLImageKHR replayCreateExternalImage(u32 rendererID, TextureId texture) { return ? }
 
+// replayFrameDelimiter
```

Comment:
This comment, is, well... empty :) Either expand it to include a message or remove it.

Reply:
Good catch, thanks!